README.md

```diff
@@ -1,26 +1,37 @@
 # Ec-Europa theme
 Repository containing the drupal theme for the NextEuropa platform.
 
-The Ec-Europa theme is a Drupal 7 theme, the one currently used by the info website (ec.europa.eu/info).
-This theme is responsive (3 different breakpoints), it uses B.E.M naming convention, in integrates a procedure to generate a styleguide and it is based on a component driven design. 
+The Ec-Europa theme is a Drupal 7 theme, implementing the styling defined for 
+the Digital Transformation of the European Commission.
+This theme is based on a component driven design. 
 
 ### 1. Installation
 
 Place the content of this repository into a folder in sites/all/themes and enable the theme going to admin/appearance.
-Install the needed libraries by running the included drush make file (drush make --no-core europa.make) 
-The Ec-Europa-theme uses bootstrap as the base theme, so you need to have also this theme, you can get it here: https://www.drupal.org/project/bootstrap
+The Ec-Europa-theme uses atomium as the base theme, so you need to have also 
+this theme, you can get it here: https://www.drupal.org/project/atomium .
```

Comment:
@Fefaine: What do you think to change this part by : "_The Ec-Europa-theme IS A sub-theme of Atomium as the base theme.
So you need to install it too.You can get it here: https://www.drupal.org/project/atomium._"